auth/request/auth_user.go

```diff
@@ -15,8 +14,8 @@ type (
 	authUser struct {
 		User *types.User
 
-		PermSession  bool
-		PermLifetime time.Duration
+		// is user logged-in with "remember-me"?
+		PermSession bool
```

Comment:
Rename all perm * to remember me for consistency?

Reply:
considered this but... no, remember-me has more of a frontend touch...